xml/System/Environment+SpecialFolder.xml

```diff
@@ -319,7 +319,7 @@
       </ReturnValue>
       <MemberValue>46</MemberValue>
       <Docs>
-        <summary>The file system directory that contains documents that are common to all users. This special folder is valid for Windows NT systems, Windows 95, and Windows 98 systems with Shfolder.dll installed. Added in the .NET Framework 4.</summary>
+        <summary>The file system directory that contains documents that are common to all users. This special folder is valid for Windows NT systems with Shfolder.dll installed.</summary>
```

Comment:
"Windows NT systems with Shfolder.dll installed" encompasses all in-market versions of Windows.